packages/apps/src/SideBar/index.tsx

```diff
@@ -190,19 +191,17 @@ class SideBar extends React.PureComponent<Props> {
   }
 
   private renderWiki () {
-    return null;
-
-    // disabled for now, we need the space
-    // return (
-    //   <Menu.Item className='apps--SideBar-Item'>
-    //     <a
-    //       className='apps--SideBar-Item-NavLink'
-    //       href='https://github.com/w3f/Web3-wiki/wiki/Polkadot'
-    //     >
-    //       <Icon name='book' /> Wiki
-    //     </a>
-    //   </Menu.Item>
-    // );
+    return (
```

Comment:
Could you revert this change? I think it‘s part of your PR #1020

Reply:
oops! Thank you